### DISCUSSION TITLE
New v6 patch

### DISCUSSION THREAD

**Discussion by marcio-secoda** [mantinedev/mantine#6434] [Q&A] [answered]
Hi team,

First of all, thank you for this amazing project!

I was wondering if we could have another patch for the v6 version. We haven't had the resources to migrate to v7 yet and we would really like to get this latest commit in: https://github.com/mantinedev/mantine/commit/6b214a5a20afc1d48127a1939514191a061352da

Please let me know if there's anything I could help to get this out :)

**Comment by rtivital** [comment, marked as answer]
Okay, I'll see if it can be published

**Reply by marcio-secoda** [reply]
@rtivital any progress getting a new patch for v6? thanks again for all your support :)

**Reply by rtivital** [reply]
The patch 6.0.22 has been published
